### PR TITLE
Fix/remove ota for tv os apps

### PR DESCRIPTION
**Pull request by tilmanx**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -9,6 +9,7 @@ gem 'rest-client', '~> 1.7.2'
 
 group :test, :development do
   gem 'rake'
+  gem 'shotgun'
```
Comment (garciaf) [inline Gemfile]: What is the purpose of shotgun?

File: shorty.rb [hunk 2]
```diff
@@ -5,37 +5,72 @@
...
       @bundle_identifier = data['bundle_identifier']
+      @platform = data['platform']
       erb :manifest
     else
       response.code
     end
   end
 end
+
+def get_image_for(platform='i')
```
Comment (garciaf) [inline shorty.rb]: Create a function for this purpose seems a little bit overkill

File: Gemfile (hunk 1 above)
Comment (tilmanx) [reply Gemfile]: shotgun automatically reloads your Sinatra app. This is a bless while developing!

File: shorty.rb (hunk 2 above)
Comment (tilmanx) [reply shorty.rb]: this is the grey line between dry and overkill

**Pull request merged**